core/controllers/learner_dashboard.py

```diff
@@ -189,6 +200,18 @@ class LearnerDashboardFeedbackThreadHandler(base.BaseHandler):
     """Gets all the messages in a thread."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'thread_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': r'\w[.]\w+[.]\w+'
```

Comment:
```suggestion
                    'regex_pattern': r'[exploration|collection]\.\w+\.\w+'
```
but add more values that are allowed. @kevintab95 @aks681 Do you know what should be the allowed values here?

Reply:
updated regex_pattern ```'(exploration|collection)\.\w+\.\w+'```, the suggested one will select only a char but I want entire string.
Please take a look.

Reply:
Yeah, right.